### PR TITLE
feat: Add support for IDMS DML commands #704

### PR DESCRIPTION
**Pull request by mpicchioni**

### CODE REVIEW

**Review by ap891843 — APPROVED** [review]
looks good to me.

File: server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolParser.g4 [hunk 1]
```diff
@@ -1073,13 +1074,39 @@ cancelCall
 // change priority statement
 
 changePriorityStatement
-    : CHANGE PRIORITY TO? (changePriorityLiteral | generalIdentifier)
+    : CHANGE PRIORITY TO? (numericLiteral | generalIdentifier)
     ;
 
 changePriorityLiteral
```
Comment (ap891843) [inline server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolParser.g4:1080]: We might get rid of this.

Comment (mpicchioni) [reply server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolParser.g4:1080]: The references to changePriorityLiteral and priorityLiteral have been removed, but I still left the items. I can remove them and re-submit the PR, or remove them in my future PR, whichever you prefer.

Comment (ap891843) [reply server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolParser.g4:1080]: Doesn't impact. I guess we can get rid of these in future PR.

**Review by grianbrcom — APPROVED** [review]
:palm_tree:

**Pull request merged**